spyder_terminal/terminalplugin.py

```diff
@@ -22,23 +22,27 @@
 
 # from spyder.preferences import PluginConfigPage
 
-from spyder.config.base import _
+from spyder.config.base import get_translation
 from spyder.utils import icon_manager as ima
 from spyder.utils.programs import find_program
 from spyder.utils.qthelpers import (add_actions, create_action,
                                     create_toolbutton,
                                     MENU_SEPARATOR)
 from spyder.widgets.tabs import Tabs
+from spyder.config.manager import CONF
```

Comment:
Please sort the import like

from spyder.config.base import get_translation
from spyder.config.manager import CONF
...